platform/qemu/virt/config.mk

```diff
@@ -30,12 +21,18 @@ PLATFORM_SYS_CLINT=y
 # Blobs to build
 FW_TEXT_START=0x80000000
 FW_JUMP=y
-# This needs to be 4MB alligned for 32-bit support
```

Comment:
You're keeping the alignment but removing the comment. Even better would be a comment as to why 32-bit has different alignment requirements, and why we don't just use the same alignment on 64-bit too.

Reply:
I removed the comment because we have detailed comments in one place at platform/template/config.mk. If you still insist I will retain the comment here.

The alignment requirement comes from the Linux RISC-V MMU bootstrap logic requirements to work correctly on 32-bit and 64-bit systems. Unfortunately, these requirements are still undocumented in Linux RISC-V.